postgresql/shell-scripts/unix/create-db-changelog.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
I prefer renaming the script to `create-changelog`. The "create-db" suggests that an entire 3DCityDB plus the changelog is installed. So it's confusing for a user.

Reply:
updated with [f85d574](https://github.com/3dcitydb/3dcitydb/pull/226/commits/f85d57478a4343404e944d8b1d2a5231ff47bfe4)